### PR TITLE
add project base menu

### PR DESCRIPTION
**Pull request by garciamendes**
Dash-menu

### CODE REVIEW

**Comment by jotasenati** [comment]
For me this alteration what Gabriel suggested that's ok !!! In outher moment we can give more suggested.

File: frontend/src/components/menu/side-menu.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+          routers.map((route, index) => {
+            if (route.menu)
+              return (
+                <MenuItem key={index}>
+                  {
+                    route.path ? (
+                      <Link to={route.path}>{route.icon && route.icon}</Link>
+                    ) : (
+                        <span>{route.icon && route.icon}</span>
```
Comment (gabrieldev525) [inline frontend/src/components/menu/side-menu.js:27]: Fix indententation

File: frontend/src/components/menu/styles.js [hunk 2]
```diff
@@ -2,19 +2,48 @@ import styled from 'styled-components'
 
 const menuHeight = 50
 
+//
```
Comment (gabrieldev525) [inline frontend/src/components/menu/styles.js:5]: remove empty comment or write any

File: frontend/src/components/menu/styles.js [hunk 3]
```diff
@@ -2,19 +2,48 @@ import styled from 'styled-components'
...
 `
+export const ImgUser = styled.p`
+  background: blue;
+  width: 35px;
+  height: 35px;
+  margin-right: 3%;
+  border-radius: 50%;
+`
+
+//
```
Comment (gabrieldev525) [inline frontend/src/components/menu/styles.js:46]: Remove empty comments or write any here

**Pull request merged**